### ISSUE TITLE
Throws error on invalid output formatter

### ISSUE THREAD

**Issue by pdehaan** [nodesecurity/gulp-nsp#13]
Here's my gulpfile (Note the invalid output formatter):

``` js
var path = require('path');
var gulp = require('gulp');
var gulpNSP = require('gulp-nsp');

// To check your project
gulp.task('nsp', function (done) {
  gulpNSP({
    output: 'summaryz',
    package: path.join(__dirname, 'package.json')
  }, done);
});
```

Result:

``` sh
$ gulp nsp

[20:04:04] Using gulpfile ~/dev/gulp-nsp-test/gulpfile.js
[20:04:04] Starting 'nsp'...
/Users/pdehaan/dev/gulp-nsp-test/node_modules/gulp-nsp/index.js:25
    var output = formatter(err, data);
                 ^
TypeError: undefined is not a function
    at /Users/pdehaan/dev/gulp-nsp-test/node_modules/gulp-nsp/index.js:25:18
    at /Users/pdehaan/dev/gulp-nsp-test/node_modules/gulp-nsp/node_modules/nsp/lib/check.js:214:7
    at /Users/pdehaan/dev/gulp-nsp-test/node_modules/gulp-nsp/node_modules/nsp/node_modules/wreck/lib/index.js:472:20
    at finish (/Users/pdehaan/dev/gulp-nsp-test/node_modules/gulp-nsp/node_modules/nsp/node_modules/wreck/lib/index.js:326:16)
    at wrapped (/Users/pdehaan/dev/gulp-nsp-test/node_modules/gulp-nsp/node_modules/nsp/node_modules/hoek/lib/index.js:866:20)
    at onReaderFinish (/Users/pdehaan/dev/gulp-nsp-test/node_modules/gulp-nsp/node_modules/nsp/node_modules/wreck/lib/index.js:363:16)
    at g (events.js:199:16)
    at emit (events.js:129:20)
    at finishMaybe (_stream_writable.js:484:14)
    at endWritable (_stream_writable.js:493:3)


$ echo $?
1
```

**Comment by pdehaan** [comment]
Ref: https://github.com/nodesecurity/grunt-nsp/pull/12

**Comment by evilpacket** [comment]
Yes I haven't ported that code over here yet :)

**Comment by joeycozza** [comment]
This should be taken care of now with this PR [https://github.com/nodesecurity/gulp-nsp/pull/15](https://github.com/nodesecurity/gulp-nsp/pull/15) which was merged.

**Comment by joeycozza** [comment]
So if an invalid formatter is given, should it just warn about it, and use the default formatter, or should it throw a gulp error saying that the given formatter is invalid?

**Comment by evilpacket** [comment]
I think it should just warn and use the default. That's what I went with int he grunt-nsp module.

**Comment by joeycozza** [comment]
Okay, I'll make that change right now.

**Comment by joeycozza** [comment]
This should be done and done now with [https://github.com/nodesecurity/gulp-nsp/pull/17](https://github.com/nodesecurity/gulp-nsp/pull/17)

**Comment by pdehaan** [comment]
Closing. Fixed by #17
